CHANGELOG.rst

```diff
@@ -1,5 +1,57 @@
 .. _changelog:
 
+0.31.0
+------
+
+Renku ``0.31.0`` introduces the compute resource control (CRC) service, enabling Renku administrators to manage access to 
+specific computing resources. The service exposes an HTTP API for the administrators to interact with. In this way, 
+Renku administrators can create, update or delete resource pools, and can add to or remove users from resource pools. 
+Please note that a user interface for the CRC has not been added yet but it will be added in a future release. Currently,
```

Comment:
I think that administrators especially the ones deploying or updating to a new version of Renku are most likely to read the changelog in the repo or the docs. At least from my experience helping admins deploy Renku I have heard complaints specifically because the release notes excluded information or were too brief. Currently we have no other place to include this information as far as I know. So I think we should definitely include this here.

Reply:
Alternatively we can make another changelog of section of the docs for this information specifically tailored for admins. But I am not sure it is worth the effort and it will only scatter information that I think belongs together.